### PR TITLE
moira 4.0.0.3 (new formula)

### PR DESCRIPTION
**Pull request by leee**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

---

Signed-off-by: Eugene Lee leee@mit.edu

### CODE REVIEW

File: Formula/moira.rb [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+                            "--with-hesiod=#{Formula["hesiod"].opt_prefix}",
+                            "--without-zephyr",
+                            "--without-oracle",
+                            "--without-afs",
+                            "--disable-rpath",
+                            "--mandir=#{man}"
+      system "make"
+      system "make", "install"
+    end
+    cd '#{bin}' do
```
Comment (ilovezfs) [inline Formula/moira.rb]: `cd bin do`

File: Formula/moira.rb [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+      mv "chfn", "chfn.moira"
+      mv "chsh", "chsh.moira"
+    end
+    cd '#{man1}' do
```
Comment (ilovezfs) [inline Formula/moira.rb]: `cd man1 do`

File: Formula/moira.rb [hunk 3]
```diff
@@ -0,0 +1,45 @@
+class Moira < Formula
+  desc "Clients for the Athena Service Management System"
+  homepage "https://debathena.mit.edu"
+  url "https://github.com/mit-athena/moira/archive/4.0.0.3+51+g65d55c5.tar.gz"
+  sha256 "d418681ae4ec61a124c55fb67a6bba0d89b59dc760c4d1c9a1e8e1c9c7b69b19"
+
+  depends_on "e2fsprogs" # for com_err, as recommended by Debian packaging
+  depends_on "hesiod"
+  depends_on "homebrew/dupes/krb5"
```
Comment (ilovezfs) [inline Formula/moira.rb]: can't depend on dupes or other taps

**Comment by ilovezfs** [comment]
```
  * GitHub repository not notable enough (<20 forks, <20 watchers and <50 stars)
```

File: Formula/moira.rb (hunk 1 above)
Comment (leee) [inline Formula/moira.rb]: Done.

File: Formula/moira.rb (hunk 2 above)
Comment (leee) [inline Formula/moira.rb]: Done.

File: Formula/moira.rb (hunk 3 above)
Comment (leee) [inline Formula/moira.rb]: Fixed, by using options. Would this be acceptable?

> can't depend on dupes or other **taps**

Wait, so you're saying I can't use homebrew-core either!? :^)

**Comment by leee** [comment]
>   * GitHub repository not notable enough (<20 forks, <20 watchers and <50 stars)
Very much a rule of thumb and a warning, not a hard blocker. Additionally, this software is provided elsewhere upstream, but is packaged for Debian using the release also used in this Formula.

**Comment by zmwangx** [comment]
> Very much a rule of thumb and a warning, not a hard blocker.

Sorry, we have to cut the line somewhere.

> is packaged for Debian using the release also used in this Formula.

I checked https://debathena.mit.edu. Third party apt repo != packaged by Debian, it contributes nothing to notability. Unless you can otherwise demonstrate notability, we'll pass on this.

**Comment by MikeMcQuaid** [comment]
Sorry about that. Please consider [creating your own tap](https://github.com/Homebrew/brew/blob/master/docs/How-to-Create-and-Maintain-a-Tap.md#how-to-create-and-maintain-a-tap). Thanks!

**Pull request closed without merging**